src/pages/pupils/beta/programmes/[programmeSlug]/units/[unitSlug]/lessons.tsx

```diff
@@ -5,14 +5,20 @@ import curriculumApi2023 from "@/node-lib/curriculum-api-2023";
 import { LessonListingBrowseData } from "@/node-lib/curriculum-api-2023/queries/pupilLessonListing/pupilLessonListing.schema";
 import getPageProps from "@/node-lib/getPageProps";
 import { resolveOakHref } from "@/common-lib/urls";
-import { URLParams } from "@/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons";
```

Comment:
We should discourage generic names like URLParams as they lead people into anti-patterns like this one.